spec/globus/client_spec.rb

```diff
@@ -83,35 +83,32 @@
     end
   end
 
-  describe "#mkdir" do
+  context "when invoking client methods" do
     before do
       allow(described_class::Endpoint).to receive(:new).and_return(fake_endpoint)
     end
 
-    let(:fake_endpoint) { instance_double(described_class::Endpoint, mkdir: nil, set_permissions: nil) }
+    let(:fake_endpoint) { instance_double(described_class::Endpoint, mkdir: nil, set_permissions: nil, file_count: nil, total_size: nil) }
     let(:user_id) { "mjgiarlo" }
     let(:work_id) { 1234 }
     let(:work_version) { 2 }
 
     it "invokes mkdir on the Endpoint class, injecting config" do
       client.mkdir(user_id:, work_id:, work_version:)
-      expect(fake_endpoint).to have_received(:mkdir).once
+      described_class.mkdir(user_id:, work_id:, work_version:)
```

Comment:
It looks like these tests are now invoking the instance and the class method in the test, whereas before we were testing them individually. Am I reading this right?

Reply:
Yes, but aside from the calls, the entire tests were duplicates. So I wanted to clean up the duplication,  I'm not entirely sure both calls are even required to be tested but coverage went down a bit.